spec/ruby-enum/enum_spec.rb

```diff
@@ -7,6 +7,17 @@ class Colors
   define :GREEN, 'green'
 end
 
+# A test class that includes CONSTANT_CASE with underscores
```

Comment:
Can I nitpick on this test? Why do we need `X11Colors` at all and can't write it all with the existing `Colors` structure? Less code, more readable, no?